app/views/sephora_style_guide/home/_accordion.slim

```diff
@@ -34,6 +34,42 @@ article
           .card-block
             | The Dursleys had everything they wanted, but they also had a secret, and their greatest fear was that somebody would discover it. They didn’t think they could bear it if anyone found out about the Potters. Mrs. Potter was Mrs. Dursley’s sister, but they hadn’t met for several years; in fact, Mrs. Dursley pretended she didn’t have a sister, because her sister and her good-for-nothing husband were as unDursleyish as it was possible to be. The Dursleys shuddered to think what the neighbors would say if the Potters arrived in the street. The Dursleys knew that the Potters had a small son, too, but they had never even seen him. This boy was another good reason for keeping the Potters away; they didn’t want Dudley mixing with a child like that.
 
+    br
+    h3.h3 Code Snippet
+
+    h4.h4 Pug
+    pre
+      code.language-pug
+        |
+          #default-accordion.accordion(aria-multiselectable='true' role='tablist')
+
+            .card.card-primary
+              #heading-one.card-header.cursor(role='tab' data-toggle='collapse' data-target='#collapse-one' aria-expanded="true" aria-controls="collapse-one")
+                h3.h3.mb-0
+                  | SIGN IN / REGISTER
+
+              #collapse-one.collapse.show(aria-labelledby='heading-one' role='tabpanel' data-parent='#default-accordion')
+                .card-block
+                  | Mr. and Mrs. Dursley, of number four, Privet Drive, were proud to say that they were perfectly normal, thank you very much. They were the last people you’d expect to be involved in anything strange or mysterious, because they just didn’t hold with such nonsense.
```

Comment:
Usually the code snippet should serve as copy-pastable base which doesn't need much changes (apart from additions of course), so placeholder text should be limited, or show exactly what needs to change, for example use keywords like `CHANGEME`).